finder-app/finder.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# Assignment 1 script for finding matching lines in a file
+# Author: Malcolm McKellips
+
+if [ $# -lt 2 ]
+then
+	echo "Not enough arguments supplied to finder script! Expected 2, got $#."
```

Comment:
@malcolmmckellips what other information could you provide to help the user here?